src/components/ChannelList/hooks/listeners/useChannelHidden.js

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect } from 'react';
+
+import { ChatContext } from '../../../../context';
+
+export const useChannelHidden = ({ onChannelHidden, setChannels }) => {
+  const { client } = useContext(ChatContext);
+
+  useEffect(() => {
+    const handleEvent = (e) => {
+      if (onChannelHidden && typeof onChannelHidden === 'function') {
```

Comment:
made this comment in few places
```suggestion
      if (typeof onChannelHidden === 'function') {
```